src/py42/_internal/clients/storage/storagenode.py

```diff
@@ -0,0 +1,46 @@
+from py42.clients import BaseClient
+import requests
+
+
+class StoragePreservationDataClient(BaseClient):
+
+    _base_uri = u"c42api/v3/"
+
+    def __init__(self, session):
+        super(StoragePreservationDataClient, self).__init__(session)
+
+    def get_download_token(self, archive_guid, file_id, timestamp):
+        """Get PDS download token of a file.
```

Comment:
for a file, since I assume the token is not identical upon each request, or is it?